softlayer/vm/softlayer_virtual_guest_deleter_test.go

```
@@ -58,13 +70,45 @@ var _ = Describe("SoftlayerVirtualGuestDeleter", func() {
 			})
 		})
 
+		Context("when can not find virtual guest with id 1234567", func() {
```

I'd expect to see a fail tests for when not found and when err...